README.md

```diff
@@ -49,8 +49,8 @@ suspend fun Payment.awaitSuccess(): Unit = throw RuntimeException("Payment Faile
 
 suspend fun main() {
   saga {
-    val order = saga { createOrder() }.compensate(::deleteOrder).bind()
-    val payment = saga { createPayment(order) }.compensate(::deletePayment).bind()
+    val order = saga(action = { createOrder() }, compensation = { ::deleteOrder })
+    val payment = saga(action = { createPayment(order) }, compensation = { ::deletePayment) })
```

Comment:
Is this syntax broken? 😱 I should've set up compile time checked docs ><

Reply:
When using version 0.1.3 the syntax was as documented but when I upgraded to 1.0.1 it changed to
`  @SagaDSLMarker
  public suspend fun <A> saga(
    action: suspend SagaActionStep.() -> A,
    compensation: suspend (A) -> Unit,
  ): A`